client/components/LocalePicker.scss

```diff
@@ -1,9 +1,25 @@
 .c_locale-picker {
   &__option {
     cursor: pointer;
+    color: #0872B9;
+    -webkit-appearance: none;
+    appearance: none;
+    background: none;
+    border: none;
+    outline: none;
+    font-weight: bold;
+    padding: 0;
 
-    &.disabled {
-      cursor: none;
+    &:not(:last-child)::after {
```

Comment:
is there an easier way to explain what this is doing and when?

Reply:
I can add a comment.